great_expectations/core/expectation_configuration.py

```diff
@@ -928,7 +928,8 @@ def __init__(
         ensure_json_serializable(meta)
         self.meta = meta
         self.success_on_last_run = success_on_last_run
-        self._ge_cloud_id = ge_cloud_id
+        if ge_cloud_id is not None:
+            self._ge_cloud_id = ge_cloud_id
```

Comment:
🙇🏼

Reply:
How does this work if `ge_cloud_id is None`? Does this not get serialized?